plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -883,7 +891,7 @@ else if (strings.size() != 2) {
         return viewToken;
     }
 
-    private static class TableToken
+    static class TableToken
```

Comment:
move to other commit (if this change is needed)

Reply:
Reverted this change.